src/Framework/Microsoft.Internal.Aspnetcore.DotNetApiDocs.Transport/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>net8.0</Target>
+  </Suppression>
```

Comment:
You better disable package validation for this project. It doesn't really provide much benefit...

Reply:
Done